mmv1/products/logging/Metric.yaml

```diff
@@ -246,6 +247,7 @@ properties:
               - bucket_options.0.exponential_buckets.0.scale
             description: |
               Must be greater than 0.
+            required: true
           - !ruby/object:Api::Type::Double
             name: growthFactor
             at_least_one_of:
```

Comment:
We should remove the requirement for `at_least_one_of` for all 3 of the child fields `numFiniteBuckets`, `growthFactor`, and `scale`

Reply:
This is why the build fails, the two restrictions (required, at least one of) are conflicting